src/components/public-service-details-page/index.tsx

```diff
@@ -152,13 +151,13 @@ const PublicServiceDetailsPage: FC<Props> = ({
   const holdsRequirement = publicService?.holdsRequirement ?? [];
   const follows = publicService?.follows ?? [];
   const hasLegalResource = publicService?.hasLegalResource ?? [];
-  const hasParticipation = publicService?.hasParticipation ?? [];
   const hasInput = publicService?.hasInput ?? [];
   const hasChannel = publicService?.hasChannel ?? [];
   const hasCost = publicService?.hasCost ?? [];
   const processingTime = publicService?.processingTime;
   const relation = publicService?.relation || [];
   const contactPoints = publicService?.contactPoint || [];
```

Comment:
Vil dette fungere også her kasnkje, bare for å holde det i sync med linjene ovenfor?
```suggestion
  const contactPoints = publicService?.contactPoint ?? [];
```

Reply:
Da bør det vel gjøres på linje 158 også. Men skal vi vente med prodsettingen, eller heller lage en ny PR med endringene du foreslår?

Reply:
@terjesyl hehe commit'et akkurat dette